common.blocks/select/select.js

```diff
@@ -60,8 +60,10 @@ provide(BEMDOM.decl(this.name, /** @lends select.prototype */{
 
             '' : function() {
                 this
-                    .unbindFromDoc('keydown', this._onKeyDown)
-                    .unbindFromDoc('keypress', this._onKeyPress)
+                    .unbindFrom('button', {
```

Comment:
Non-blocker: It'd be nice to place this about `_focus` (as `_unfocus` mb). Just for readability.